jenkins-examples/job-dsl-examples/pipeline-example/JobDSL.groovy

```diff
@@ -7,7 +7,7 @@ pipelineJob('job-dsl-artifactory-pipeline-example') {
 
     definition {
         cps {
-            script(readFileFromWorkspace('jenkins-examples/pipeline-examples/declarative-example/Jenkinsfile'))
+            script(readFileFromWorkspace('jenkins-examples/pipeline-examples/scripted-examples/declarative-example/Jenkinsfile'))
```

Comment:
Was this path changed intentionally? The path looks wrong (...scripted-examples/declarative-example...)

Reply:
Yes, it is an example of a scripted pipeline implemented as a step in a declarative pipeline.
The other path does not exist